alpha-build-machinery/make/targets/openshift/images.mk

```diff
@@ -1,25 +1,32 @@
-IMAGE_REGISTRY ?=
-IMAGE_ORG ?=openshift
-IMAGE_TAG ?=latest
-
-
 # IMAGE_BUILD_EXTRA_FLAGS lets you add extra flags for imagebuilder
 # e.g. to mount secrets and repo information into base image like:
 # make images IMAGE_BUILD_EXTRA_FLAGS='-mount ~/projects/origin-repos/4.2/:/etc/yum.repos.d/'
+IMAGE_BUILD_DEFAULT_FLAGS ?=--allow-pull
 IMAGE_BUILD_EXTRA_FLAGS ?=
 
-# $1 - image name
-# $2 - Dockerfile path
-# $3 - context
+# $1 - target name
+# $2 - image registry
+# $3 - image org
+# $4 - image name
+# $5 - image tag
+# $6 - Dockerfile path
+# $7 - context
 define build-image-internal
 image-$(1):
-	$(strip imagebuilder --allow-pull $(IMAGE_BUILD_EXTRA_FLAGS) -f $(2) -t $(addsuffix /,$(IMAGE_REGISTRY))$(addsuffix /,$(IMAGE_ORG))$(1)$(addprefix :,$(IMAGE_TAG)) $(3))
+	$(strip \
```

Comment:
what does this add to calling imagebuilder directly?

Reply:
strips white chars one space